tests/src/rules/no-extraneous-dependencies.js

```diff
@@ -243,24 +248,23 @@ ruleTester.run('no-extraneous-dependencies', rule, {
       options: [{packageDir: packageDirMonoRepoWithNested}],
       errors: [{
         ruleId: 'no-extraneous-dependencies',
-        message: "'left-pad' should be listed in the project's dependencies. Run 'npm i -S left-pad' to add it",
+        message: '\'left-pad\' should be listed in the project\'s dependencies. Run \'npm i -S left-pad\' to add it',
       }],
     }),
     test({
       code: 'import react from "react";',
       filename: path.join(packageDirMonoRepoRoot, 'foo.js'),
       errors: [{
         ruleId: 'no-extraneous-dependencies',
-        message: "'react' should be listed in the project's dependencies. Run 'npm i -S react' to add it",
+        message: '\'react\' should be listed in the project\'s dependencies. Run \'npm i -S react\' to add it',
       }],
     }),
     test({
-      code: 'import react from "react";',
```

Comment:
if this test case is supposed to pass, let's move it to "valid"

Reply:
I solved this by:
- keeping this test. since it's also good to be checking
- added exact original test to valid, here https://github.com/benmosher/eslint-plugin-import/pull/1104/files#diff-05140578b7a943405919d8a058934d1dR97